Task/MCU/Floor1_USART_inputdata/Core/Src/stm32f4xx_it.c

```diff
@@ -209,8 +208,7 @@ void USART2_IRQHandler(void)
   /* USER CODE END USART2_IRQn 0 */
   HAL_UART_IRQHandler(&huart2);
   /* USER CODE BEGIN USART2_IRQn 1 */
-
-	  inter();
+  inter();
```

Comment:
Put this function at receive interrupt code

Reply:
done